### DISCUSSION TITLE
Add Events Calendar for 2021 in our GitHub Organisation

### DISCUSSION THREAD

**Discussion by vinzvinci** [EddieHubCommunity/support#604] [Ideas] [answered]
I think its cool if we have **Events Calendar for this 2021 in our GitHub Organization**. Everyone can add events if they want to share, for example hacktoberfest, other virtual events and etc. :)

What do you think?

**Comment by eddiejaoude** [comment]
This is a great idea 👍 - we could start off simple with a markdown file? Could we add it to an existing repo or is a new repo needed? What does everyone think?

**Reply by eddiejaoude** [reply]
I pinned this to the discussion section

<img width="1663" alt="Screenshot 2021-01-03 at 20 08 10" src="https://user-images.githubusercontent.com/624760/103487737-75020d00-4dff-11eb-8c83-48c5ea3c4db1.png">

**Reply by Vyvy-vi** [reply]
I think we could use this repo(a new file called `CALENDAR.md` perhaps?)
We could make some neat markdown/html tables for each month

**Reply by Vyvy-vi** [reply]
OR we could also use this discussion to hold the calendar, but IMHO that might lead to issues, since maybe someone mis-interprets those due to a comment or something

**Reply by eddiejaoude** [reply]
I suppose the history in the file will get big 🤔 . Could we use 1 discussion per month and people can add threads for each event? Not sure if a good idea or not, just throwing it out there 🤓

**Comment by Akshu-on-github** [comment]
Why don't we create a GitHub pages site for this?

**Reply by vinzvinci** [reply]
I agree with markdown and GitHub pages.

**Reply by Vyvy-vi** [reply]
yes, that would be good :+1:

**Reply by eddiejaoude** [reply]
Yep, markdown and GitHub pages 👍

**Reply by manavmodi22** [reply]
How about a GitHub Actions integrated here?

**Comment by eddiejaoude** [comment]
Also might be good to add TwitterSpaces events to this list too (maybe with a type)

**Comment by ghost** [comment]
My words are to make a google calendar add it to our support repo, it will allow us a ton of new features like google will notify people for every upcoming events, things like twitter spaces need to be edited frequently and google will help us in that too

**Reply by Vyvy-vi** [reply]
We could do that, tho then only Eddie could add events == too much work for Eddie. Perhaps we could make a GitHub action, that can also read events and time from a file, and adds it to the calendar whenever a PR is made to alter that file

**Comment by vinzvinci** [comment]
## My suggestion:
https://demo.vercel.events/

## Update:
We will use the same tool we used to build the website (Kartra, low code solution), so it integrates with the rest of our tools.
`-` by @eddiejaoude  

Just putting it here so Eddie won't forget this

**Comment by vinzvinci** [comment, marked as answer]
Now existing: https://github.com/EddieHubCommunity/EventCalendar

Please pin the ambassador program @eddiejaoude and remove this discussion to pinned. Thanks :)

**Reply by eddiejaoude** [reply]
Sure will do 👍 The ambassador program is coming along, we can't wait to share it very soon 🎉
